lib/twilio_voice.dart

```diff
@@ -259,7 +262,8 @@ class Call {
 
   /// Gets the active call's SID. This will be null until the first Ringing event occurs
   Future<String?> getSid() {
-    return _channel.invokeMethod<String?>('call-sid', <String, dynamic>{}).then<String?>((String? value) => value);
+    return _channel.invokeMethod<String?>('call-sid',
```

Comment:
@agent515 lets get rid of these changes please, i.e. only the error code changes.

Reply:
@cybex-dev I understand what you're saying. But I think it's linter that is doing these changes upon saving file. I'm not sure about how to prevent this.

Reply:
No worries, thanks